string  name
string  status

bool    ready
float32 rate  # [hz]
